partials/_machines_nav.html.erb

```diff
@@ -7,16 +7,41 @@
 
 <ul class="outline-list">
   <li>
+    <span>
+      Control Machines with flyctl
+    </span>
+  <ul>  
+   <li>
     <%= nav_link "Run a new Machine", "/docs/machines/run/" %>
-  </li>
+   </li>
+   <li>
+    <%= nav_link "Update a Machine", "/docs/machines/run/" %>
+   </li>
```

Comment:
```suggestion
```

I'm actively drafting a m update doc but there isn't one to link to yet, right?

Reply:
no! I removed this for now, but I want to keep the section